handler/src/test/java/io/netty/handler/ssl/JdkSslEngineTest.java

```diff
@@ -263,6 +267,40 @@ public String select(List<String> protocols) {
         }
     }
 
+    @Test
+    public void testEnablingAnAlreadyDisabledSslProtocol() throws Exception {
```

Comment:
can we just have this test live in `SSLEngineTest`? That way we won't need to duplicate the code.